nipype/algorithms/misc.py

```diff
@@ -262,6 +262,11 @@ class TSNRInputSpec(BaseInterfaceInputSpec):
     in_file = InputMultiPath(File(exists=True), mandatory=True,
                              desc='realigned 4D file or a list of 3D files')
     regress_poly = traits.Range(low=1, desc='Remove polynomials')
+    tsnr_file = File('tsnr.nii.gz', usedefault=True, desc='output tSNR file')
```

Comment:
actually could we use `name_source` and `name_template` here? i'm not sure though whether they work for all types of interfaces. hopefully then do.